From 2bee9bb3de43d421e89a4a0e71a07dbecd059475 Mon Sep 17 00:00:00 2001
From: Chirayu Desai <chirayudesai1@gmail.com>
Date: Sun, 28 Aug 2016 23:02:27 +0530
Subject: [PATCH 029/249] Revert "Add missing projects" - We want Browser

* Browser2 is just a minimal test browser,
  Browser is the full featured version and has been updated
  by us.

This reverts commit 2115a47430dac5dc90184084eb2dbab98bb83351.

Change-Id: I92ecdccfee79c8da7befb25771a23d29b123446d
---
 default.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/default.xml b/default.xml
index e96f7ec..469e329 100644
--- a/default.xml
+++ b/default.xml
@@ -191,7 +191,7 @@
   <project path="packages/apps/BasicSmsReceiver" name="CyanogenMod/android_packages_apps_BasicSmsReceiver" groups="pdk-cw-fs,pdk-fs" />
   <project path="packages/apps/Bluetooth" name="CyanogenMod/android_packages_apps_Bluetooth" groups="pdk-cw-fs,pdk-fs" />
   <project path="packages/apps/BluetoothExt" name="CyanogenMod/android_packages_apps_BluetoothExt" />
-  <project path="packages/apps/Browser2" name="CyanogenMod/android_packages_apps_Browser2" groups="pdk-fs" />
+  <project path="packages/apps/Browser" name="CyanogenMod/android_packages_apps_Browser" groups="pdk-fs" />
   <project path="packages/apps/Calendar" name="CyanogenMod/android_packages_apps_Calendar" groups="pdk-fs" />
   <project path="packages/apps/Camera2" name="CyanogenMod/android_packages_apps_Camera2" groups="pdk-fs" />
   <project path="packages/apps/CarrierConfig" name="CyanogenMod/android_packages_apps_CarrierConfig" groups="pdk-fs" />
@@ -535,6 +535,7 @@
   <project path="hardware/ti/omap4-aah" name="platform/hardware/ti/omap4-aah" groups="omap4-aah" remote="aosp" />
   <project path="libnativehelper" name="platform/libnativehelper" groups="pdk" remote="aosp" />
   <project path="ndk" name="platform/ndk" groups="generic_fs" remote="aosp" />
+  <project path="packages/apps/Browser2" name="platform/packages/apps/Browser2" remote="aosp" />
   <project path="packages/apps/OneTimeInitializer" name="platform/packages/apps/OneTimeInitializer" remote="aosp" />
   <project path="packages/services/Car" name="platform/packages/services/Car" groups="adp8064,pdk-cw-fs,pdk-fs" remote="aosp" />
   <project path="pdk" name="platform/pdk" groups="pdk" remote="aosp" />
-- 
2.17.1

